legal/index.md

```diff
@@ -53,15 +53,15 @@ By posting or submitting any material to this website, including on the forum, y
+The Monero Project takes many precautions to ensure the files available for downloading through this website are not tampered with, and where appropriate, [SHA hashes](https://www.getmonero.org/downloads/hashes.txt) are provided for end users to confirm they have downloaded the correct file. Nonetheless, The Monero Project does not guarantee that files available for downloading through this website will be free of infection, viruses or other code that may have contaminating or destructive properties. You are responsible for implementing sufficient procedures and checkpoints to satisfy your particular requirements for integrity, security and accuracy of data input and output, and for maintaining a means external to the site for the reconstruction of any lost data. It is almost always advisable to check the SHA hashes of the files you have downloaded, and if in any doubt, consider [compiling Monero from the source code](https://github.com/monero-project/monero#compiling-monero-from-source).
 
 ### Sponsors and Advertisers are Not Endorsed
 
 Though we seek to partner with quality companies, we are not responsible for the claims or representations made by sponsors and advertisers. The Monero Project does not review, endorse or recommend the products or services of any company mentioned on our site. We will not be liable in any way for damages of any kind resulting out of the misuse of any personal information or data submitted by you to a sponsor or advertiser.
 
 ### Offsite Links
 
-As a convenience to Users, the The Monero Project website and services may provide links to other sites or resources. Because we do not review or have no control over such sites and resources, The Monero Project shall not be responsible or liable for use of or reliance on any content, products, services or information at such sites or resources. Inclusion of any links does not imply any endorsement, affiliation, approval, association or sponsorship by The Monero Project of the linked websites, resources, their operators or owners. When you select a link, you may be leaving our website. The information available on Third Parties' websites may have certain restrictions on its use or distribution which differ from these TOU.
+As a convenience to Users, the The Monero Project website and services may provide links to other sites or resources. Because we do not review or have no control over such sites and resources, The Monero Project shall not be responsible or liable for the use of, or reliance on any content, products, services or information at such sites or resources. Inclusion of any links does not imply any endorsement, affiliation, approval, association or sponsorship by The Monero Project of the linked websites, resources, their operators or owners. When you select a link, you may be leaving our website. The information available on Third Parties' websites may have certain restrictions on its use or distribution which differ from these TOU.
```

Comment:
```suggestion
As a convenience to Users, The Monero Project website and services may provide links to other sites or resources. Because we do not review or have no control over such sites and resources, The Monero Project shall not be responsible or liable for the use of, or reliance on any content, products, services or information at such sites or resources. Inclusion of any links does not imply any endorsement, affiliation, approval, association or sponsorship by The Monero Project of the linked websites, resources, their operators or owners. When you select a link, you may be leaving our website. The information available on Third Parties' websites may have certain restrictions on its use or distribution which differ from these TOU.
```

Same on L68